cpp/src/feather/CMakeLists.txt

```diff
@@ -21,6 +21,16 @@ set(LIBFEATHER_SRCS
   status.cc
   types.cc
   writer.cc
+  api.h
+  buffer.h
+  common.h
+  feather-c.h
+  io.h
+  metadata.h
+  reader.h
+  status.h
+  types.h
+  writer.h
```

Comment:
Can you explain this? Is this something required to make `WINDOWS_EXPORT_ALL_SYMBOLS` work?